services/stats/testdata/otel-collector-config.yaml

```diff
@@ -0,0 +1,24 @@
+receivers:
```

Comment:
**Question:** is this an alternative way for configuring otel? Are we really using it?

Reply:
That is the agent/collector configuration. That configuration that you see there in the `otel-collector-config.yaml` file is not what we'll use in production of course. However some things are similar like the receivers. We'll have a single receiver getting metrics/traces over the gRPC connection on 4317, which is this bit:

```yaml
receivers:
  otlp:
    protocols:
      grpc:
```

If that doesn't answer your question please let me know as I'm unsure about what you were asking exactly.

Reply:
Thanks, I confused collectors with exporters for a while